File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomManagerContacter/DbAtomManagerContacterStatusEnum.cs
namespace DataModelLibrary.Database.AtomManagerContacter;

/// <summary> 資料庫-原子-管理者-聯絡人狀態-列舉 </summary>
public enum DbAtomManagerContacterStatusEnum : short
{
    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>不清楚(預設)</summary>
    Unknown = 1,

    /// <summary>在職</summary>
    Employed = 2,

    /// <summary>離職</summary>
    Unemployed = 3,
}

File: FrontEnd/manager-back-site/src/constants/DbAtomManagerContacterStatusEnum.ts
/** 資料庫-原子-管理者-聯絡人狀態-列舉 */
export enum DbAtomManagerContacterStatusEnum {
  // /** 未選擇 */
  // NotSelected = -1,
  /** 未定義 */
  Undefined = 0,
  /** 不清楚(預設) */
  Unknown = 1,
  /** 在職 */
  Employed = 2,
  /** 離職 */
  Unemployed = 3
}